lib/portlayer/exec/exec_test.go

```diff
@@ -29,32 +29,32 @@ import (
 
 var containerEvents []events.Event
 
-func TestEventedState(t *testing.T) {
-	// poweredOn event
-	event := events.ContainerPoweredOn
-	assert.EqualValues(t, StateStarting, eventedState(event, StateStarting))
-	assert.EqualValues(t, StateRunning, eventedState(event, StateRunning))
-	assert.EqualValues(t, StateRunning, eventedState(event, StateStopped))
-	assert.EqualValues(t, StateRunning, eventedState(event, StateSuspended))
-
-	// powerOff event
-	event = events.ContainerPoweredOff
-	assert.EqualValues(t, StateStopping, eventedState(event, StateStopping))
-	assert.EqualValues(t, StateStopped, eventedState(event, StateStopped))
-	assert.EqualValues(t, StateStopped, eventedState(event, StateRunning))
-
-	// suspended event
-	event = events.ContainerSuspended
-	assert.EqualValues(t, StateSuspending, eventedState(event, StateSuspending))
-	assert.EqualValues(t, StateSuspended, eventedState(event, StateSuspended))
-	assert.EqualValues(t, StateSuspended, eventedState(event, StateRunning))
-
-	// removed event
-	event = events.ContainerRemoved
-	assert.EqualValues(t, StateRemoved, eventedState(event, StateRunning))
-	assert.EqualValues(t, StateRemoved, eventedState(event, StateStopped))
-	assert.EqualValues(t, StateRemoving, eventedState(event, StateRemoving))
-}
+// func TestEventedState(t *testing.T) {
```

Comment:
Why is this commented out?  If it shouldn't just be removed, then use t.Skip("Reason") instead of commenting out.